### ISSUE TITLE
bug: AggregateMinMaxFunction' result is wrong when there is a NULL value.

### ISSUE THREAD

**Issue by RinChanNOWWW** [datafuselabs/databend#7413]
### Search before asking

- [X] I had searched in the [issues](https://github.com/datafuselabs/databend/issues) and found no similar issues.

### Version

v0.7.66-nightly-e6c565e-simd(rust-1.64.0-nightly-2022-08-31T06:25:42.547873538Z)

### What's Wrong?

```
mysql> select * from t;
+------+------+
| a    | b    |
+------+------+
| NULL |    1 |
|    1 |    1 |
|    2 |    2 |
|    3 |    2 |
|    3 |    3 |
|    4 |    3 |
| NULL |    4 |
| NULL |    4 |
+------+------+
mysql> select min(a) from t;
+--------+
| min(a) |
+--------+
|      0 |
+--------+
mysql> select min(a), b from t group by b;
+--------+------+
| min(a) | b    |
+--------+------+
|      0 |    1 |
|      0 |    4 |
|      3 |    3 |
|      2 |    2 |
+--------+------+
```

### Are you willing to submit PR?

- [x] Yes I am willing to submit a PR!

**Comment by RinChanNOWWW** [comment]
cc @sundy-li

**Comment by RinChanNOWWW** [comment]
It's because the bitmap the function gets is wrong.

**Comment by RinChanNOWWW** [comment]
https://github.com/datafuselabs/databend/blob/7c769f0adee771f3aaf8e1ed42706257ce2165dc/src/query/functions/src/aggregates/aggregate_scalar_state.rs#L118-L129

It is because the validity information in `viewer` is worng. We should use `bit` to judge the validity.

**Comment by RinChanNOWWW** [comment]
And we should `remove_nullable` for `min`, `max` (and `any`) functions:

https://github.com/datafuselabs/databend/blob/7c769f0adee771f3aaf8e1ed42706257ce2165dc/src/query/functions/src/aggregates/adaptors/aggregate_null_unary_adaptor.rs#L106-L118

**Comment by sundy-li** [comment]
Fixed in #7412
